### PR TITLE
Adding EOL Notice for Legacy Feature

### PR DESCRIPTION
**Pull request by nateprice7**
Adding End of Life notice for the 1.3 APIs, SOAP, and Analytics oAuth

## Description

This PR is providing notice of the end of life of some legacy portions of our APIs. These items have been deprecated for some time and  the time has come to provide notice and turn these features off. 

## Checklist:

- [x ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ x] My code follows the code style of this project.
- [ x] My change requires a change to the documentation.
- [ x] I have updated the documentation accordingly.
- [x ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ x] All new and existing tests passed.

### CODE REVIEW

**Comment by mike2184** [comment]
Looks good to me

File: docs/APIEOL.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## Soap Authentication
+
+### What Are These Services?
+
+The 1.3 and 1.4 APIs have historically supported soap. Soap is not very common now days and the vast majority of our users are using our rest interfaces. Due to the lack of interest in our SOAP interfaces it is being retired at the same time as the 1.3 APIs.
+
+### What do I Need to do to Migrate?
+
+You will Since Soap is no longer supported you will need to use the 1.4 API rest interface (https://github.com/AdobeDocs/analytics-1.4-apis). Depending on the language you are using this may necessitate finding a new rest client library.
```
Comment (stevja1) [inline docs/APIEOL.md]: Get rid of 'You will' at the beginning of this sentence.

File: docs/APIEOL.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## Legacy oAuth
+
+### What Are These Services?
+
+Legacy oAuth is an authentication service that was provided by Adobe Analytics before AdobeID and was an option. It was used by Adobe's customers to authenticate to a number of Adobe Services and in some cases for Adobe Services to communicate to each other. Now that AdobeID is widely available we are retiring the Analytics Legacy oAuth solution in favor of AdobeID.
+
+### What do I Need to do to Migrate?
+
+In general we are using AdobeID authentication as a replacement for legacy oAuth. Depending on  your particular use case the steps to adopt AdobeID will be
```
Comment (stevja1) [inline docs/APIEOL.md]: There is some funny white space (spaces and new lines) in this paragraph.

File: docs/APIEOL.md (hunk 1 above)
Comment (nateprice7) [reply docs/APIEOL.md]: Done

File: docs/APIEOL.md (hunk 2 above)
Comment (nateprice7) [reply docs/APIEOL.md]: I should have that fixed now

**Pull request merged**